src/XFExpandableListView/Cells/ExpandableGroupCell.cs

```diff
@@ -64,58 +64,59 @@ protected override void OnChildAdded(Element child)
 
             _holderAdded = true;
 
-            var holder = new Grid();
-            holder.Children.Add(view);
-            holder.GestureRecognizers.Add(
-                new TapGestureRecognizer
+            View.GestureRecognizers.Add(new TapGestureRecognizer
+            {
+                Command = new Command(async () =>
                 {
-                    Command = new Command(async () =>
+                    #region [Toggle Group]
+
+                    /* If is collapsing enabled, then collapse/expand the group */
+                    if (ExpandableController.IsCollapsingEnabled)
                     {
-                        #region [Toggle Group]
+                        await ExpandableController.ToggleGroup(GroupController);
+                    }
 
-                        /* If is collapsing enabled, then collapse/expand the group */
-                        if (ExpandableController.IsCollapsingEnabled)
-                        {
-                            await ExpandableController.ToggleGroup(GroupController);
-                        }
+                    #endregion
 
-                        #endregion
+                    #region [Execute Cell Command]
 
-                        #region [Execute Cell Command]
+                    /* Execute the cell command */
+                    if (CellCommand == null) return;
```

Comment:
Made some changes here in the meantime. Try pull and commit the changes again :)

Reply:
Merge was a bit incorrect, fixed that